ambry-api/src/main/java/com/github/ambry/config/ClusterMapConfig.java

```diff
@@ -342,6 +346,18 @@ public class ClusterMapConfig {
   @Default("false")
   public final boolean clustermapEnableDeleteInvalidDataInMysqlAggregationTask;
 
+  /**
+   * True to enable property store clean up task.
+   */
+  @Config(ENABLE_PROPERTY_STORE_CLEAN_UP_TASK)
+  public final boolean clustermapEnablePropertyStoreCleanUpTask;
+
+  /**
+   * True to remove data from datanode config when property store clean up task is enabled.
+   */
+  @Config(DELETE_DATA_FROM_DATANODE_CONFIG_IN_PROPERTY_STORE_CLEAN_UP_TASK)
+  public final boolean clustermapDeleteDataFromDatanodeConfigInPropertyStoreCleanUpTask;
```

Comment:
Provide a default false value 

`@Default("false")`

Reply:
I have provided default value already in the constructor.